internal/util/metricsinfo/container_test.go

```diff
@@ -12,24 +12,38 @@
 package metricsinfo
 
 import (
+	"runtime"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
+func isLinux() bool {
```

Comment:
you can use xxx_linux.go and add "// +build linux"  into the file to run this test only Linux platform